runtime/development/src/lib.rs

```diff
@@ -1556,6 +1556,8 @@ frame_support::parameter_types! {
 	#[derive(scale_info::TypeInfo, Debug, PartialEq, Clone)]
 	pub const MaxChangesPerEpoch: u32 = 50;
 
+	pub const InitialEpochDuration: BlockNumber = prod_or_fast!(5 * MINUTES, 1 * MINUTES);
```

Comment:
Since this is the `development-runtime`, I would have a default of 1 * MINUTE only, without the `prod_or_fast` annotation. Since in most cases that is quick enough already to verify anything with a test. In most of the cases the regularly built wasm (no fast-runtime feature) is quick enough on most of the session, epochs etc cycles, so it doesnt need to use the `prod_or_fast`.
The `prod_or_fast` is more useful when testing on production runtimes like altair & centrifuge.

Reply:
Makes sense!